### PR TITLE
CDAP-5108 Publish access audit only when audit publisher is not null

### PR DESCRIPTION
**Pull request by poornachandra**
Build - http://builds.cask.co/browse/CDAP-DUT3685-2

### CODE REVIEW

File: cdap-data-fabric/src/main/java/co/cask/cdap/data2/audit/AuditPublishers.java [hunk 1]
```diff
@@ -28,15 +30,19 @@
...
    * @param entityId entity id for which audit information is being published
    * @param accessType access type
    * @param accessor the entity accessing entityId
    */
-  public static void publishAccess(AuditPublisher publisher, EntityIdCompatible entityId, AccessType accessType,
-                                   EntityIdCompatible accessor) {
+  public static void publishAccess(@Nullable AuditPublisher publisher, EntityIdCompatible entityId,
+                                   AccessType accessType, EntityIdCompatible accessor) {
+    if (publisher == null) {
+      return;
```
Comment (sreevatsanraman) [inline cdap-data-fabric/src/main/java/co/cask/cdap/data2/audit/AuditPublishers.java]: Log a Warning? Since this shouldn't be ideally null.

**Comment by sreevatsanraman** [comment]
Minor comment. Fix and merge. LGTM

**Comment by sreevatsanraman** [comment]
lgtm

**Comment by poornachandra** [comment]
Build is green, merging.

**Pull request merged**